src/go/k8s/kuttl-v2-test.yaml

```diff
@@ -12,10 +12,10 @@ commands:
   - command: kubectl taint nodes -l node-role.kubernetes.io/master= node-role.kubernetes.io/master:NoSchedule-
     ignoreFailure: true
   - command: ./hack/install-cert-manager.sh tests/_e2e_artifacts_v2
-  - command: bash -c "kubectl kustomize https://github.com/redpanda-data/redpanda//src/go/k8s/config/crd | kubectl apply -f -"
```

Comment:
What was the problem?

Reply:
When using this approach, we are tied to using crds based on dev, so my PR, for example on a backport branch would fail the tests because the crds did not match the changes being made in the backport itself. This should prevent that.